.github/workflows/go-presubmit.yml

```
@@ -119,6 +119,37 @@ jobs:
           KUBECONFIG: /home/runner/.kube/config
 
+  e2e-token:
+    name: e2e-token
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+      - name: install Go
+        uses: actions/setup-go@v3
```

actionlint (1.7.10)

the runner of "actions/setup-go@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)